CSS/quizzshow.css

```diff
@@ -85,6 +85,105 @@
     background-color: #991b1b;
 }
 
+.btn--light{
+    background-color: var(--clr-light);
+    color: var(--clr-primary);
+}
+
+.btn--light:hover{
+    background-color: var(--clr-light);
+}
+
+.rules__container{
+    max-width: 50%;
+}
+
+.rules{
+    min-height: 70vh;
+}
+
+.question{
+    min-height: 70vh;
+}
+
+.question__container{
+    max-width: 60%;
+}
+
+.result__answer{
+    max-width: 100%;
+}
+
+.container__answer{
+    margin-top: var(--margin-large);
+    display: flex;
+    flex-direction: column;
+}
+
+.answer{
+    padding: var(--padding-large);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid grey;
+    border-radius: 10px;
+    margin: var(--margin-medium) 0;
+    cursor: pointer;
+    background-color: var(--clr-primary);
+}
+
+.answer:hover{
+    background-color: var(--clr-primary-dark);
+}
+
+.answer--success{
+    background-color: #15803d;
+    border: none;
+}
+
+.answer--success:hover{
+    background-color: #15803d;
+    border: none;
+}
+
+.answer--wrong{
+    background-color: #dc2626;
+    border: none;
+}
+
+.answer--wrong:hover{
+    background-color: #dc2626;
```

Comment:
you using same color on hover everywhere, also use variables.

Reply:
yes thank you will create css colors for success and error, hover color is same for success and wrong as i didn't want hover effect to work when answer shown